packages/manager/apps/ips/src/pages/listing/ipListing/components/DatagridCells/IpReverse/IpReverse.tsx

```diff
@@ -20,20 +25,104 @@ export type IpReverseProps = {
  */
 export const IpReverse = ({ ip, parentIpGroup }: IpReverseProps) => {
   const { expiredIps } = useContext(ListingContext);
+  const { addError } = useNotifications();
 
   // Check if ip is not /32
   const { ip: formattedIp, isGroup } = ipFormatter(ip);
+  const ipGroup = parentIpGroup || ip;
 
   // Get ip reverse
   const { ipsReverse, isLoading, error } = useGetIpReverse({
-    ip: parentIpGroup || ip,
+    ip: ipGroup,
     enabled: !isGroup && expiredIps.indexOf(ip) === -1,
   });
 
+  // Prepare delete mutation with
+  // ip: single Ip with mask (ipFormatter remove mask if /32)
+  // ipGroup: the parent group for Ip (if /32 ipGroup is same as Ip but with the mask)
+  const {
+    mutate: deleteIpReverse,
+    isPending: pendingDelete,
+    error: deleteIpReverseError,
+    isError: isDeleteIpReverseError,
+  } = useDeleteIpReverse({
+    ip: formattedIp,
+    ipGroup,
+  });
+
+  // Prepare update mutation with
+  // ip: single Ip with mask (ipFormatter remove mask if /32)
+  // ipGroup: the parent group for Ip (if /32 ipGroup is same as Ip but with the mask)
+  const {
+    mutate: updateIpReverse,
+    isPending: pendingUpdate,
+    error: updateIpReverseError,
+    isError: isUpdateIpReverseError,
+  } = useUpdateIpReverse({
+    ip: formattedIp,
+    ipGroup,
+  });
+
+  // If there is an error while deleting / updating
+  // Add error notification
+  useEffect(() => {
+    if (isUpdateIpReverseError || isDeleteIpReverseError) {
+      const apiError = updateIpReverseError || deleteIpReverseError;
+      addError(
+        <Translation ns="error">
+          {(_t) => (
+            <span
+              dangerouslySetInnerHTML={{
+                __html: _t('managerApiError', {
+                  error:
+                    apiError?.response?.data?.message ||
+                    apiError?.message ||
+                    null,
+                  requestId: apiError?.response?.headers?.['x-ovh-queryid'],
+                }),
+              }}
+            />
+          )}
+        </Translation>,
+        true,
+      );
+    }
+  }, [
+    isUpdateIpReverseError,
+    isDeleteIpReverseError,
+    updateIpReverseError,
+    deleteIpReverseError,
+  ]);
+
```

Comment:
It's better to pass a function to onError callback of the mutation inside `useUpdateIpReverse`